### DISCUSSION TITLE
d1 mini switch

### DISCUSSION THREAD

**Discussion by lukaszwitkowski** [RavenSystem/esp-homekit-devices#1818] [MEPLHAA Scripts]
hello i have d1 mini witch motion sensor humidity and temperature . 2 switch and 2 relay

haw I can change switch from on/off to click version in code

{
  "c": { "l": 13 },
  "a": [{
    "t": 1,
    "0": { "r": [{ "g": 0, "v": 0  }] },
    "1": { "r": [{ "g": 0, "v": 1 }] },
    "b": [{"g":5,"p":1,"t":1},{"g":5,"t":0}],
    "s": 5
  },
  {
    "t": 1,
    "0": { "r": [{ "g": 3, "v": 0  }] },
    "1": { "r": [{ "g": 3, "v": 1 }] },
    "b": [{"g":4,"p":1,"t":1},{"g":4,"t":0}],
    "s": 5
  },
  {
    "t":12,"i":10, 	"0":{"r":[{"g":2,"v":0}]},"1":{"r":[{"g":2,"v":1}]},
    "f1":[{"g":16}]},
{
    "t":30,
    "ty":8,
    "g":[1,1,20]
  },
{
    "t": 24,
    "g": 14,
    "n": 2,
    "j": 30,
    "z": -0.8
  }]
}

**Comment by lukaszwitkowski** [comment]
now I press button and he stay and I want to have momentary switch what I need to change in button line?

**Reply by pawelosin** [reply]
Jeżeli masz włącznik chwilowy i po naciśnięciu wraca on do pozycji wyjściowej, tak jak włącznik dzwonkowy, to zmień w konfiguracji przycisku na "b": [{"g":5,"p":1}] i analogicznie przy drugim przycisku. Ze względu na to że "t":1 jest domyślne nie musi się pojawiać w zapisie konfiguracji.
